ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -167,9 +169,14 @@ class ConfirmPaymentMethodScreen extends React.Component<Props, never> {
                 <Col size={9}>
                   <View spacer={true} large={true} />
                   <Text style={WalletStyles.textCenter}>
-                    {`${I18n.t("wallet.ConfirmPayment.info2")} `}
-                    <Text link={true}>
-                      {I18n.t("wallet.ConfirmPayment.changeMethod")}
+                    {/* TODO: the proper UI needs to be defined for changing PSP */}
+                    {this.props.wallet.psp !== undefined
+                      ? `Your current psp is ${
```

Comment:
translate

Reply:
This was intended to be a temporary placeholder while waiting for the finalized UI version but I guess we can keep it as is now and have the finalized UI be implemented at a later time -- I'll add the internationalized version